### PR TITLE
Powerfist code improvement

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
I wanted to add some nuance to the used gasmix in the powerfist but i want to do some cleanups first so here we are, i've also added some more text in examine and item use for a bit of quality of life

## Why It's Good For The Game
cleaner code, better examine for players

## Changelog

:cl:
code: cleaned and brought up to date the powerfist code
qol: added some more examine text on the powerfist
/:cl:

### CODE REVIEW

File: code/game/objects/items/powerfist.dm [hunk 1]
```diff
@@ -27,51 +35,53 @@
...
-		if(1)
-			fisto_setting = 2
-		if(2)
-			fisto_setting = 3
-		if(3)
-			fisto_setting = 1
+	fist_pressure_setting = fist_pressure_setting >= HIGH_PRESSURE ? LOW_PRESSURE : fist_pressure_setting + 1
 	tool.play_tool_sound(src)
-	to_chat(user, span_notice("You tweak \the [src]'s piston valve to [fisto_setting]."))
+	balloon_alert(user, "you tweak \the [src]'s piston valve to [fist_pressure_setting]")
```
Comment (JohnFulpWillard) [inline code/game/objects/items/powerfist.dm:45]:
```suggestion
	balloon_alert(user, "set to [fist_pressure_setting]")
```
balloon alerts are meant to be short and simple.

File: code/game/objects/items/powerfist.dm [hunk 2]
```diff
@@ -27,51 +35,53 @@
...
 	return TRUE
 
 /obj/item/melee/powerfist/screwdriver_act(mob/living/user, obj/item/tool)
-	if(tank)
-		updateTank(tank, 1, user)
-		return TRUE
+	if(!tank)
+		balloon_alert(user, "no tank present")
+		return
+	update_tank(tank, TRUE, user)
```
Comment (JohnFulpWillard) [inline code/game/objects/items/powerfist.dm:52]: can you specify what TRUE means here?

**Review by JohnFulpWillard — COMMENTED** [review]
Only problems I've got, looks good otherwise.

File: code/game/objects/items/powerfist.dm [hunk 3]
```diff
@@ -109,10 +124,16 @@
...
 
-	target.throw_at(throw_target, 5 * fisto_setting, 0.5 + (fisto_setting / 2))
+	target.throw_at(throw_target, 5 * fist_pressure_setting, 0.5 + (fist_pressure_setting / 2))
 
 	log_combat(user, target, "power fisted", src)
 
 	user.changeNext_move(CLICK_CD_MELEE * click_delay)
 
-	return
+	our_turf.assume_air(gas_used)
```
Comment (Watermelon914) [inline code/game/objects/items/powerfist.dm:133]: this seems like an addition, yet it isn't included anywhere in your PR.

Comment (Watermelon914) [reply code/game/objects/items/powerfist.dm:133]: I see

**Pull request merged**